src/components/StudentGrid.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import ButtonBase from '@material-ui/core/ButtonBase';
+import logopic from '../img/logo.png';
+import LoginButton from './LoginButton';
+import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  paper: {
+    padding: theme.spacing(2),
+    margin: 'auto',
+  },
+  image: {
+    width: 750,
+    height: 250,
+  },
+  img: {
+    margin: 'auto',
+    display: 'block',
+    maxWidth: '100%',
+    maxHeight: '100%',
+  },
+}));
+
+export default function StudentGrid() {
```

Comment:
I suggest combining this with CommunityGrid as they're very similar and just taking in props for the things that are different (title, description, image). You can call this something like `ScholarshipsMadeSimpleGrid`

Reply:
Still relevant though now I think the combo should be called `ScholarshipsMadeSimpleTab`

Reply:
Ok I still what you're talking about merge the two files and change the display based on the props given, let me give it a shot

Reply:
File and function name changed now I will figure out how to take in a parameter input to be used for showing different content